File: src/main/java/org/awesome/leetcode/algorithm/array/TwoSum.java
package org.awesome.leetcode.algorithm.array;

public class TwoSum {
    public int[] twoSum(int[] nums, int target) {
        int[] result = new int[2];
        for (int i = 0; i < nums.length; i++) {
            for (int j = i+1; j < nums.length; j++) {
                if(target == (nums[i]+nums[j])){
                    result[0] = i;
                    result[1] = j;
                }
            }
        }
        return result;
    }
}

File: main/two-sum.go
package main

import (
	"fmt"

	"awesome.leetcode.algorithm/leetcode"
)

func main() {
	//输入：nums = [2,7,11,15], target = 9
	//输出：[0,1]
	nums := []int{2, 7, 11, 15}
	target := 9

	res := leetcode.TwoSum(nums, target)
	fmt.Println("res = ", res)
}
